packages/core/src/blueprint-modern.scss

```diff
@@ -0,0 +1,548 @@
+/*
+
+Copyright 2015-present Palantir Technologies, Inc. All rights reserved.
+Licensed under the Apache License, Version 2.0.
+
+*/
+
+// Import files in the same order that they are documented in the docs
+@import "common/variables-modern";
+@import "~@blueprintjs/colors/lib/css/colors.css";
+
+@import "reset";
+@import "typography";
+@import "accessibility/focus-states";
+
+// Button variables and overrides
```

Comment:
Some of these variables override the existing defaults, e.g. `$button-intents`. Others are just new variables that I needed to make e.g. `$button-active-minimal-intent-text-modern`

Reply:
sounds good to me